src/django/api/middleware.py

```
@@ -132,3 +136,44 @@ def __call__(self, request):
 
         response = self.get_response(request)
         return response
+
+
+class DarkVisitorsMiddleware:
```

Pylint (3.3.7)

[refactor] Too few public methods (1/2)

(R0903)